src/views/dashboard/components/threadFeed.js

```diff
@@ -304,6 +304,7 @@ class ThreadFeed extends React.Component<Props, State> {
           useWindow={false}
           initialLoad={false}
           scrollElement={scrollElement}
+          showPTRBefore={'#scroller-for-inbox'}
```

Comment:
@mxstbr we receive selector before where we should show pullToRequest element.  what do you say ??

Reply:
@mxstbr what are other places we need pullToRequest other than `threadFeed`

Reply:
No other places we need to have pull to refresh, for now just the thread feed is good enough.